proxy/src/utils/fetchPageData.ts

```diff
@@ -54,8 +54,12 @@ export const parseRawContent = ({
     ...(metaFromMarkdown as object),
   }
 
+  const formattedTitle = title.includes('nordcraft')
+    ? title.replace('nordcraft', 'Nordcraft')
+    : title
```

Comment:
Consider this instead:
```suggestion
  const formattedTitle = title.replaceAll('nordcraft', 'Nordcraft')
```
Also, when is this relevant?

Reply:
It looks like it's relevant here:

![image](https://github.com/user-attachments/assets/a6f1453a-d4f6-479a-bd06-4d775128e737)

I originally thought it could be a front end issue and that the titles were sentence cased, but it looks like the meta data coming in had a lowercase n, and I think that's how the sidebar is populated.

I think this is worth keeping in, but perhaps a change also needs to be done on the `fetchMenuItems`.

Reply:
Yes I've found it